static/app/views/performance/landing/widgets/widgets/lineChartListWidget.tsx

```diff
@@ -251,158 +402,32 @@ export function LineChartListWidget(props: PerformanceWidgetProps) {
             <SelectableList
               selectedIndex={selectedListIndex}
               setSelectedIndex={setSelectListIndex}
-              items={provided.widgetData.list.data.map(listItem => () => {
-                const transaction = (listItem.transaction as string | undefined) ?? '';
-
-                const additionalQuery: Record<string, string> = {};
-
-                if (props.chartSetting === PerformanceWidgetSetting.SLOW_HTTP_OPS) {
-                  additionalQuery.breakdown = 'http';
-                  additionalQuery.display = 'latency';
-                } else if (props.chartSetting === PerformanceWidgetSetting.SLOW_DB_OPS) {
```

Comment:
moved `items` calculation logic to its function `getItems` since this logic is duplicated across two menus